test/components/button/button.puppeteer-spec.js

```diff
@@ -33,5 +33,25 @@ describe.skip('Button Puppeteer Tests', () => {
       expect(await page.$eval('#btn-primary-target', el => getComputedStyle(el).getPropertyValue('height'))).toMatch('34px');
     });
   });
+
+  describe('Button notifBadge test', () => {
```

Comment:
```suggestion
  describe('Button notifBadge test', () => {
```

Not sure what was meant here ? Notification badge?

Reply:
yep, will update this as "Button Notification Badge toggle test"